src/Octoshift/Commands/ReclaimMannequin/ReclaimMannequinCommandArgs.cs

```diff
@@ -20,5 +21,10 @@ public override void Validate(OctoLogger log)
         {
             throw new OctoshiftCliException($"Either --csv or --mannequin-user and --target-user must be specified");
         }
+
+        if (string.IsNullOrEmpty(Csv) && SkipInvitation)
```

Comment:
Is there a reason why you can only "skip" with a CSV? That seems like a surprising limitation.

Reply:
The original batch lists the scenario as specifically using the generated CSV. 

Skimming through, I don't see an immediate reason as to why we can't use it for just the single reclamation on the CLI side of things.